source/arborx/access_traits.cc

```diff
@@ -146,6 +146,69 @@ namespace ArborXWrappers
   {
     return n_nearest_neighbors;
   }
+
+  // ------------------- SpherePredicate ------------------- //
+  template <int dim, typename Number>
+  SpherePredicate::SpherePredicate(
+    const std::vector<std::pair<dealii::Point<dim, Number>, Number>>
+      &dim_spheres)
```

Comment:
Is there a particular reason why you use the name `dim_spheres` here but just `spheres` elsewhere? Just curious...

Reply:
Ah, I see. Thanks a lot for the explanation!